README.md

```diff
@@ -2,39 +2,296 @@
+None
+
+## GET Field Definitions 
+### `GET /definition/context/:apiContext/version/:apiVersion`
+Retrieves the definitions of subscription fields for an API
+
+### Response with
+
+| Status | Description                          |
+|--------|--------------------------------------|
+| 200    | OK                                   |
+| 404    | Not found                            |
+
+### GET Field Definitions example
+
+#### CURL command
+```
+curl -v -X GET "http://localhost:9650/definition/context/hello/version/1.0" -H "Cache-Control: no-cache"
+```
+#### Response body
+```json
+{
+  "fieldDefinitions": [
+    {
+      "name": "callback-url",
+      "description": "Callback URL",
+      "type": "URL"
+    },
+    {
+      "name": "token",
+      "description": "Secure Token",
+      "type": "SecureToken"
+    }
+  ]
+}
+```
+
+## GET Field Definitions For All APIs
+### `/definition`
+Retrieves the definitions of subscription fields for all APIs
+
+### Response with
+
+| Status | Description                          |
+|--------|--------------------------------------|
+| 200    | OK                                   |
+| 404    | Not found                            |
+
+### GET Field Definitions example
+
+#### CURL command
+```
+curl -v -X GET "http://localhost:9650/definition"   -H "Cache-Control: no-cache"
+```
+#### Response body
+```json
+{
+  "fieldDefinitions": [
+    {
+      "name": "callback-url",
+      "description": "Callback URL",
+      "type": "URL"
+    },
+    {
+      "name": "token",
+      "description": "Secure Token",
+      "type": "SecureToken"
+    }
+  ]
+}
+```
+
+## PUT Field Values 
+### `PUT /field/application/:clientId/context/:apiContext/version/:apiVersion`
+Creates or updates the field values of an API subscription   
+
+### Response with
+
+| Status | Description                          |
+|--------|--------------------------------------|
+| 201    | Created                              |
+| 200    | Updated                              |
+| 404    | Not found                            |
+
+### PUT Field Values
+
+#### CURL command
+```
+curl -v -X PUT "http://localhost:9650/field/application/hBnFo14C0y4SckYUbcoL2PbFA40a/context/hello/version/1.0" -H "Cache-Control: no-cache" -H "Content-Type: application/json" -d '{ "fields" : { "callback-url" : "http://localhost:8080/callback", "token" : "abc59609za2q" } }'
+```
+#### Request body
+```json
+{
+  "fields": {
+    "callback-id": "http://localhost",
+    "token": "abc123"
+  }
+}
+```
+
+#### Response body
+```json
+{
+  "clientId" : "xcsvvbe2882L",
+  "apiContext" : "hello",
+  "apiVersion" : "1.0",
+  "fieldsId" : "55c2b945-1c82-4749-b4fc-42e5u32192ew", 
+  "fields": {
+    "callback-id": "http://localhost",
+    "token": "abc123"
+  }
+}
 ```
 
+## GET Field Values 
+### `GET /field/application/:clientId/context/:apiContext/version/:apiVersion`
+Retrieves the field values of an API subscription by providing the application and API details
 
-### License
```

Comment:
You removed the `License` section
Could you bring it back on top of the readme?

Reply:
done